tests/unit/auth/V4Transform.js

```diff
@@ -49,11 +54,15 @@ describe('V4Transform class', () => {
             assert.strictEqual(err, null);
             done();
         });
-        const chunks = [new Buffer('8;chunk-signature=' +
-        '51d2511f7c6887907dff20474d8db67d557e5f515a6fa6a8466bb12f8833bcca\r\n' +
-        'contents\r\n'), new Buffer('0;chunk-signature=' +
-        'c0eac24b7ce72141ec077df9753db4cc8b7991491806689da0395c8bd0231e48\r\n'),
-        new Buffer('\r\n'), null];
+        const filler1 = '8;chunk-signature=51d2511f7c6887907dff20474d8db6' +
+        '7d557e5f515a6fa6a8466bb12f8833bcca\r\ncontents\r\n';
+        const filler2 = '0;chunk-signature=c0eac24b7ce72141ec077df9753db' +
+        '4cc8b7991491806689da0395c8bd0231e48\r\n';
+        const chunks = [
+            Buffer.alloc(filler1.length, filler1),
+            Buffer.alloc(filler2.length, filler2),
```

Comment:
Are we missing the third buffer from above `new Buffer('\r\n')` here?

Reply:
nice catch, ty! 👍